app/src/main/java/com/rage/clamber/Activities/LoginActivity.java

```diff
@@ -64,71 +63,107 @@ public void onExistingUserButtonClicked (Button button) {
 
     }
 
+    /**
+     * Method called from ExistingUserLoginFragment when a username is entered. The method makes a
+     * call to the server to determine if the username exists in the database. If it does, the user
+     * is logged into the application and brought to the home page. The User is stored in the Home
+     * Page class in order to pull up other data related to that User in various fragments.
+     * @param user - username that has been entered in dialog fragment.
+     */
+
     public void existingUserPositiveClick(String user) {
         userName = user;
         if (userName.length() == 0) {
-            Toast.makeText(this, "Please enter a user name", Toast.LENGTH_SHORT).show();
+            Toast.makeText(this, R.string.enter_username, Toast.LENGTH_SHORT).show();
         } else {
             //Check for internet connectivity
             ConnectivityManager connectivityManager = (ConnectivityManager) this.getSystemService(Context.CONNECTIVITY_SERVICE);
             NetworkInfo networkInfo = connectivityManager.getActiveNetworkInfo();
             if (networkInfo != null && networkInfo.isConnected()) {
-                loginAsyncTask = new LoginAsyncTask(this);
-                loginAsyncTask.execute(userName);
+                Retrofit retrofit = new Retrofit.Builder()
+                        .baseUrl(HomePage.CONNECTION_WEB_ADDRESS)
+                        .addConverterFactory(JacksonConverterFactory.create())
+                        .build();
+                ClamberService clamber = retrofit.create(ClamberService.class);
+                Call<User> userCall = clamber.getExistingUser(userName);
+
+                userCall.enqueue(new Callback<User>() {
+                    @Override
+                    public void onResponse(Call<User> call, Response<User> response) {
+
+                        User user = response.body();
+                        if (response.code() == 200) {
+                            String login = response.body().getUserName();
```

Comment:
It seems like the server should only ever return the user if the username matches what you supplied, so checking the equality on the return shouldn't be necessary. Just getting a 200 back should indicate that the user does in fact exist, unless I'm missing something about how the server operates.